src/components/MapNode.tsx

```diff
@@ -76,6 +78,40 @@ export const MapNode = observer((props) => {
     setMapDepth(depth - 1)
   }
 
+  React.useEffect(() => {
+    if (orderedPhrasings.length != 0 || !detailViewOpen) return //only do the sort once, when the detail view opens
+    phrasings.sort((a, b) => {
```

Comment:
My first choice is still with lodash, like this:
```
const {simple, standard, technical} = _.groupBy(orderedPhrasings, (phrasing) => 
    _.startsWith(phrasing, "[Simple]") ? "simple" : (
        _.startsWith(phrasing, "[Technical]") ? "technical" : "standard"
    )
)
_.concat(simple, standard, technical)
```

But I don't want to cramp your style so if you leave it like this I just thought it would be nice to make consts for whether a has Simple, a has Technical, ditto for b, at the beginning, and then do the tests.

Reply:
OK fiiiiiiiine ;-)